src/browser/ui/dom/components/ReactDOMInput.js

```diff
@@ -35,6 +37,13 @@ function forceUpdateIfMounted() {
   }
 }
 
+function isIeInputEvent(event) {
+  return ('documentMode' in document)
+      && document.documentMode > 9
+      && event.nativeEvent.type === 'input'
+      && isTextInputElement(event.target);
+}
```

Comment:
2 things here.

1. `isIEInputEvent`
2. Put the `&&` at the end of the previous lines and then let's wrap the whole return in parens

```js
return (
  'documentMode' in document &&
  document.documentMode > 9 &&
  etc
)
```

And now actually a 3rd since I'm looking. We can cache the value of `'documentMode' in document && document.documentMode > 9` so that we don't need to look it up each time (any access of DOM properties has a cost).